pages/[project]/changes_logs.vue

```diff
@@ -182,7 +182,7 @@ async function resetFilters() {
 }
 
 function matchFilterBySelectors(selectors: string[]) {
-  return route.query.filterBySelectors !== undefined && intersection(selectors, route.query.filterBySelectors as string).length > 0
+  return route.query.filterBySelectors !== undefined && intersection(selectors, [route.query.filterBySelectors as string]).length > 0
```

Comment:
not just ?
```suggestion
  return route.query.filterBySelectors !== undefined && route.query.filterBySelectors as string in selectors
```

Reply:
`in` operator is for Object not Array ([Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in))
but we could do `selectors.includes(route.query.filterBySelectors as string)` and get rid of `intersection` dependency.

I've updated the PR !